docs/source/metadata_parsing.mdx

```diff
@@ -92,40 +92,39 @@ export type SafetensorsShardedHeaders = Record<FileName, SafetensorsFileHeader>;
 
 ### Python
 
-In this example python script, we are parsing metadata of [gpt2](https://huggingface.co/gpt2/blob/main/model.safetensors).
+[`huggingface_hub`](https://huggingface.co/docs/huggingface_hub/index) provides a Python API to parse safetensors metadata.
```

Comment:
```suggestion
[`huggingface_hub`](https://huggingface.co/docs/huggingface_hub) provides a Python API to parse safetensors metadata.
```

it will automatically redirect to index page